packages/test/test-end-to-end-tests/src/test/localTestSignals.spec.ts

```diff
@@ -84,28 +84,28 @@ describeFullCompat("TestSignals", (getTestObjectProvider) => {
         it("Validate host runtime signals", async () => {
             let user1SignalReceivedCount = 0;
             let user2SignalReceivedCount = 0;
-            const user1ContainerRuntime = dataObject1.context.containerRuntime;
-            const user2ContainerRuntime = dataObject2.context.containerRuntime;
+            const user1ContainerRuntime = dataObject1.context;
+            const user2ContainerRuntime = dataObject2.context;
 
-            user1ContainerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
+            user1ContainerRuntime.containerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
                 if (message.type === "TestSignal") {
                     user1SignalReceivedCount += 1;
                 }
             });
 
-            user2ContainerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
+            user2ContainerRuntime.containerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
                 if (message.type === "TestSignal") {
                     user2SignalReceivedCount += 1;
                 }
             });
 
             user1ContainerRuntime.submitSignal("TestSignal", true);
```

Comment:
The reason this test is failing is that the signal is now being sent from the `DataStoreContext` but listened for on the `ContainerRuntime`, since the definition of `user1ContainerRuntime` was changed.